src/retrace/hooks/hooks.py

```diff
@@ -99,7 +99,7 @@ def _process_script(self, hook: str, hook_path: Path, exc_path: str) -> None:
 
         try:
             child = run(cmd, shell=True, timeout=hook_timeout, cwd=hook_path,
-                        stdout=PIPE, stderr=PIPE, encoding='utf-8')
+                        stdout=PIPE, stderr=PIPE, encoding='utf-8', check=False)
             child.check_returncode()
```

Comment:
We could just shorten this to
```suggestion
                        stdout=PIPE, stderr=PIPE, encoding='utf-8', check=True)
```
since we're calling `check_returncode()` afterwards anyway.

Reply:
Thanks, fixed.